### ISSUE TITLE
Throttles CPU down below 1.2 GHz

### ISSUE THREAD

**Issue by fghzxm** [xmrig/xmrig-nvidia#127]
Using this software causes the CPU to throttles down to 1.0~1.2 GHz even when the CPU temp is only 52 ℃. Power supply is not an issue. Why does this happen?

**Comment by xmrig** [comment]
Are you sure it's throttle? any modern CPU dynamically change clocks to reduce power usage.
Thank you.

**Comment by fghzxm** [comment]
Yes. I kept the Task Manager window open, and no matter what I do on Windows, the CPU frequency didn’t exceed 1.2 GHz, even when I do computationally intensive tasks.
Switching to “best performance” mode does mitigate this issue; now it can manage up to 2.3 GHz, still lower than its 2.8 GHz base frequency.
